src/__tests__/components/ExploreList.spec.tsx

```diff
@@ -27,7 +27,7 @@
 import React from 'react';
 import renderer from 'react-test-renderer';
 import { mockCurrentExplore, mockModels } from "../MockData/MockData";
-import { theme} from "@looker/components"
+import { theme } from "@looker/components"
```

Comment:
i'm happy you changed this. be forewarned though that some people get grumpy when you make whitespace changes to files you're not otherwise touching. grumpy old grumps.